src/improvements/tasks/sep/018-U16-remove-dgm/VALIDATION.md

```diff
@@ -7,11 +19,84 @@ the values printed to the terminal when you run the task.
 >
 > Before signing, ensure the below hashes match what is on your ledger.
 >
-> ### Nested Safe 1 (SecurityCouncil): `0xf64bc17485f0B4Ea5F06A96514182FC4cB561977`
+> ### Nested Safe 1 (Foundation): `0x837DE453AD5F21E89771e3c06239d8236c0EFd5E`
+>
+> - Domain Hash: `0xe84ad8db37faa1651b140c17c70e4c48eaa47a635e0db097ddf4ce1cc14b9ecb`
+> - Message Hash: `0xa5b581bbbdd908f535d8a3ba49d91fa41e828e735a918fe3832179e85359dfef`
```

Comment:
This task is running on the [GuardianSafe](https://github.com/ethereum-optimism/superchain-ops/blob/f883d9327ca33bae4d318c4e489b8a8ec942242f/src/improvements/tasks/sep/018-U16-remove-dgm/config.toml#L3), which is only owned by the Security Council, so we shouldn't see the Foundation here.

Reply:
Pushed a fix for this.